python/kserve/kserve/model_server.py

```diff
@@ -83,6 +84,8 @@ def __init__(self, http_port: int = args.http_port,
         self.enable_latency_logging = validate_enable_latency_logging(enable_latency_logging)
 
     def create_application(self):
+        dataplane = handlers.DataPlane(model_registry=self.registered_models)
+        self._grpc_server = GRPCServer(port=self.grpc_port, data_plane=dataplane)
```

Comment:
`create_application` is tornado specific, it is better to separate out grpc server related code.